relay-kafka/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+pub mod config;
```

Comment:
It would be great if we could use this as a chance to improve documentation. Particularly, the root doc comment of the crate is valuable because it can explain the purpose of the crate and point to the various entry-points.

Reply:
👍  I will create some initial docs here.